pkg/utils/controller/controller.go

```
@@ -86,6 +86,24 @@ func NewUserError(err error) error {
 	return nil
 }
 
+// NewCreateIgnoreAlreadyExistError returns ErrExpectedBehavior type error if the error is already exist.
+// Otherwise, returns ErrAPIServerError type error.
+func NewCreateIgnoreAlreadyExistError(err error) error {
+	if !apierrors.IsAlreadyExists(err) {
+		return NewAPIServerError(false, err)
+	}
+	return NewExpectedBehaviorError(err)
```

when it's alreadyExists, should we ignore the error?

---

nvm, should proceed to update the spec